src/nvim/ex_getln_defs.h

```diff
@@ -49,7 +49,7 @@ struct cmdline_info {
   int cmdfirstc;                ///< ':', '/', '?', '=', '>' or NUL
   int cmdindent;                ///< number of spaces before cmdline
   char *cmdprompt;              ///< message in front of cmdline
-  int cmdattr;                  ///< attributes for prompt
+  int cmdhlid;                  ///< highlight id for prompt
```

Comment:
`cmd_hl_id` (or maybe `cmdhl_id`). it is better to be consistent with always using `hl_id` elsewhere, never `hlid`